puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -49,10 +45,6 @@
 
   $pgroonga = zulipconf('machine', 'pgroonga', '')
   if $pgroonga == 'enabled' {
-    apt::ppa {'ppa:groonga/ppa':
-      before => Package["postgresql-${zulip::base::postgres_version}-pgroonga"],
-    }
```

Comment:
Similarly here, I think -- we still want to depend on `setup-apt-repo` being run.